src/compiler/crystal/command/deps.cr

```diff
@@ -1,14 +0,0 @@
-# Implementation of the `crystal deps` command, which
-# simply delegates to the `shards` executable
-
-class Crystal::Command
-  private def deps
-    path_to_shards = `which shards`.chomp
-    if path_to_shards.empty?
-      error "`shards` executable is missing. Please install shards: https://github.com/crystal-lang/shards"
```

Comment:
@anatol FYI this has obsoleted the patch in the Arch package
https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/crystal&id=3354aceae25c5b81f53cf6362ce463ef3890f398#n25